brave/brave-core
# Moves pending contributon into mojom definition

Resolves https://github.com/brave/brave-browser/issues/4620

## Submitter Checklist:

- Verified that these changes build without errors on
- Verified that these changes pass automated tests (`npm test brave_unit_tests && npm test brave_browser_tests && npm run test-security`) on

## Test Plan:
- enable rewards
- claim grant
- tip multiple unverified site
- make sure that you see pending box in your wallet
- open dialog
- try deleting single site
- try deleting all sites
- make sure that everything is carried out correctly

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on.
